classes/options-general.php

```diff
@@ -394,19 +394,19 @@ function print_page() {
 									<!-- #### MAP CONFIGURATION #### -->
 									<div class="postbox">
 
-										<h3><?php _e( 'Map Configuration', 'simplemap' ); ?></h3>
+										<h3><?php _e( 'Map Configuration', 'SimpleMap' ); ?></h3>
 
 										<div class="inside">
-											<p class="sub"><?php printf( __( 'See %s the Help page%s for an explanation of these options.', 'simplemap' ), '<a href="' . get_bloginfo( 'wpurl' ) . '/wp-admin/admin.php?page=simplemap-help">', '</a>&nbsp;' ); ?></p>
+											<p class="sub"><?php printf( __( 'See %s the Help page%s for an explanation of these options.', 'SimpleMap' ), '<a href="' . get_bloginfo( 'wpurl' ) . '/wp-admin/admin.php?page=simplemap-help">', '</a>&nbsp;' ); ?></p>
 
 											<div class="table">
 												<table class="form-table">
 
 												<tr valign="top">
-													<td width="150"><label for="api_key"><?php _e( 'Google Maps API Key', 'simplemap' ); ?></label></td>
+													<td width="150"><label for="api_key"><?php _e( 'Google Maps API Key', 'SimpleMap' ); ?></label></td>
```

Comment:
__[code_sniffer]__
* Spaces must be used for alignment; tabs are not allowed
* Space after opening parenthesis of function call prohibited
* Expected 0 spaces before closing bracket; 1 found
* Line exceeds 120 characters; contains 152 characters

<!-- Comment by [SideCI](https://sideci.com) -->